client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -275,6 +277,13 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 			this.openPreviewModal( postUrl, ports[ 0 ] );
 		}
 
+		if ( EditorActions.ViewPost === action ) {
+			const { postUrl } = payload;
+			config.isEnabled( 'desktop' )
+				? this.props.notifyDesktopViewPostClicked( postUrl )
+				: window.open( postUrl );
```

Comment:
This changes behavior of the "View Post" action for the web client. Ther original code called `window.open( url, '_top' )`, opening the post URL in the same tab (i.e., the top parent frame of the `iframe`.

Now, the mere `window.open( url )` will open a new tab.

I like the "open the new tab" behavior better, but if we change it, it should be intentional.

Reply:
@jsnajdr Thanks for taking a peek! After consulting/testing with @mmtr, we determined this should be fine. I'm also +1 for opening the post in a new tab - it feels a little bit nicer to not force the user out of their current context. That being said, if we get any feedback to the contrary, it would be trivial to change and happy to do so.